lte/gateway/python/integ_tests/s1aptests/test_no_auth_resp_with_mme_restart_reattach.py

```diff
@@ -74,7 +74,7 @@ def test_no_auth_resp_with_mme_restart_reattach(self):
         # time, MME sometimes restarts after a delay of around 6 seconds. If
         # MME restarts after T3460 timer expiry of 6 seconds, it will send
         # re-transmitted Authentication Request message
-        if response.msg_type == s1ap_types.tfwCmd.UE_AUTH_REQ_IND.value:
+        while response.msg_type != s1ap_types.tfwCmd.UE_CTX_REL_IND.value:
             print(
```

Comment:
Did you check, how a failing test would behave here? I guess, the timeout error of the `get_response()` does the trick, but we don't have any other way to break out of the while loop?

Reply:
Yes, exactly, if the test fails here it is because it runs into the timeout of `get_response()`